newsfragments/3234.bugfix

```diff
@@ -0,0 +1 @@
+Add coverage information to integration tests
```

Comment:
Or "Collect coverage information from ..."?  Maybe this is more of a feature or a misc from a user perspective (*user*-facing Tahoe-LAFS functionality isn't changing).